[2025-01-31T05:03:36.481+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_v3.third_task scheduled__2021-09-19T00:00:00+00:00 [queued]>
[2025-01-31T05:03:36.501+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_v3.third_task scheduled__2021-09-19T00:00:00+00:00 [queued]>
[2025-01-31T05:03:36.502+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-01-31T05:03:36.547+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): third_task> on 2021-09-19 00:00:00+00:00
[2025-01-31T05:03:36.574+0000] {standard_task_runner.py:57} INFO - Started process 10397 to run task
[2025-01-31T05:03:36.590+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag1_v3', 'third_task', 'scheduled__2021-09-19T00:00:00+00:00', '--job-id', '3119', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpugfs_nsi']
[2025-01-31T05:03:36.593+0000] {standard_task_runner.py:85} INFO - Job 3119: Subtask third_task
[2025-01-31T05:03:36.772+0000] {task_command.py:415} INFO - Running <TaskInstance: dag1_v3.third_task scheduled__2021-09-19T00:00:00+00:00 [running]> on host f8bdfcb8bca5
[2025-01-31T05:03:36.966+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anhtupham99' AIRFLOW_CTX_DAG_ID='dag1_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2021-09-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-09-19T00:00:00+00:00'
[2025-01-31T05:03:36.968+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-01-31T05:03:36.970+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo I am the third task, I run after the task 1, at the same time with task 2']
[2025-01-31T05:03:36.995+0000] {subprocess.py:86} INFO - Output:
[2025-01-31T05:03:36.998+0000] {subprocess.py:93} INFO - I am the third task, I run after the task 1, at the same time with task 2
[2025-01-31T05:03:36.999+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-01-31T05:03:37.050+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1_v3, task_id=third_task, execution_date=20210919T000000, start_date=20250131T050336, end_date=20250131T050337
[2025-01-31T05:03:37.095+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-31T05:03:37.162+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
